lib/emerald.rb

```diff
@@ -18,12 +18,15 @@
 module Emerald
   # The Emerald CLI
   class CLI < Thor
-    class_option :beautify, aliases: 'b'
+    class_option :beautify, :type => :boolean, :aliases => 'b'
```

Comment:
Nice catch, it should have had a type associated with it, rather than just a generic option. 👍